src/get_target_data.R

```diff
@@ -87,61 +87,71 @@ get_target_data <- function(
   output_file <- fs::path(output_dirpath, "time-series", ext = "parquet")
   hubverse_format_nhsn_data <- raw_nhsn_data |>
     dplyr::rename(
-      observation = totalconfc19newadm,
-      date = weekendingdate,
-      state = jurisdiction
+      observation = "totalconfc19newadm",
+      date = "weekendingdate",
+      state = "jurisdiction"
     ) |>
     dplyr::mutate(
-      date = as.Date(date),
-      observation = as.numeric(observation),
-      state = stringr::str_replace(state, "USA", "US")
+      date = as.Date(.data$date),
+      observation = as.numeric(.data$observation),
+      state = stringr::str_replace(.data$state, "USA", "US")
     ) |>
-    dplyr::filter(!stringr::str_detect(state, "Region")) |>
+    dplyr::filter(!stringr::str_detect(.data$state, "Region")) |>
     dplyr::mutate(
-      location = forecasttools::us_loc_abbr_to_code(state),
-      as_of = today,
+      location = forecasttools::us_location_recode(.data$state, "abbr", "code"),
+      as_of = !!today,
       target = "wk inc covid hosp"
     ) |>
-    dplyr::filter(!(location %in% excluded_locations))
+    dplyr::filter(!(location %in% !!excluded_locations))
 
   hubverse_format_nhsn_data |>
     dplyr::rename(
       value = observation
     ) |>
-    dplyr::select(-c(as_of, target)) |>
+    dplyr::select(-c("as_of", "target")) |>
     readr::write_csv(
       fs::path(output_dirpath, "covid-hospital-admissions.csv")
     )
 
   raw_nssp_data <- readr::read_csv(
-    fs::path(base_hub_path, "auxiliary-data", "nssp-raw-data", "latest.csv"),
+    fs::path(
+      base_hub_path,
+      "auxiliary-data",
+      "nssp-raw-data",
+      "latest",
+      ext = "csv"
+    ),
     show_col_types = FALSE
   )
 
   hubverse_format_nssp_data <- raw_nssp_data |>
     dplyr::filter(county == "All") |>
     dplyr::mutate(
-      date = as.Date(week_end),
-      observation = as.numeric(percent_visits_covid) / 100,
-      location = ifelse(fips == "00000", "US", stringr::str_sub(fips, 1, 2))
+      date = as.Date(.data$week_end),
+      observation = as.numeric(.data$percent_visits_covid) / 100,
+      location = ifelse(
+        .data$fips == "00000",
+        "US",
+        stringr::str_sub(.data$fips, 1, 2)
+      )
     ) |>
     dplyr::mutate(
-      state = forecasttools::us_loc_code_to_abbr(location),
-      as_of = today,
+      state = forecasttools::us_location_recode(.data$location, "code", "abbr"),
```

Comment:
This is likely better than truncating fips code?
```suggestion
      state = forecasttools::us_location_recode(.data$geography, "name", "abbr"),
      location = forecasttools::us_location_recode(.data$geography, "name", "code"),
```